js-inputs-and-strings_change-box-appearance/js/index.js

```diff
@@ -16,3 +16,29 @@ For further information check MDN:
 - about CSS declarations with "style": https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style
 
 */
+
+const colorInput = document.querySelector('[data-js="input-color"]');
+const radiusInput = document.querySelector('[data-js="input-radius"]');
+const rotationInput = document.querySelector('[data-js="input-rotation"]');
+
+const box = document.querySelector('[data-js="box"]');
+
+colorInput.addEventListener("input",(e) => {
+
+box.style.background = `hsl(${e.target.value}, 70%, 60%)`;
```

Comment:
Fix inner scope indents using 1 tab character space length and unnecessary blank lines. Same in the eventListeners below. Besides that, if it works, then it's all good